<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AmsFromFiles</class>
 <widget class="QDialog" name="AmsFromFiles">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>587</width>
    <height>183</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Расстояния по осям измерений</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>X</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="distX">
        <property name="decimals">
         <number>1</number>
        </property>
        <property name="maximum">
         <double>300.000000000000000</double>
        </property>
        <property name="value">
         <double>80.000000000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>м</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Y</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="distY">
        <property name="decimals">
         <number>1</number>
        </property>
        <property name="maximum">
         <double>300.000000000000000</double>
        </property>
        <property name="value">
         <double>80.000000000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>м</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Файлы измерений</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>X</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="fileX"/>
      </item>
      <item>
       <widget class="QPushButton" name="selectX">
        <property name="text">
         <string>Выбрать</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Y</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="fileY"/>
      </item>
      <item>
       <widget class="QPushButton" name="selectY">
        <property name="text">
         <string>Выбрать</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AmsFromFiles</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>173</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AmsFromFiles</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>173</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>selectX</sender>
   <signal>clicked()</signal>
   <receiver>AmsFromFiles</receiver>
   <slot>selectXFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>229</x>
     <y>121</y>
    </hint>
    <hint type="destinationlabel">
     <x>6</x>
     <y>137</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>selectY</sender>
   <signal>clicked()</signal>
   <receiver>AmsFromFiles</receiver>
   <slot>selectYFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>522</x>
     <y>124</y>
    </hint>
    <hint type="destinationlabel">
     <x>582</x>
     <y>137</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>selectXFile()</slot>
  <slot>selectYFile()</slot>
 </slots>
</ui>
